src/org/wikibase/Wikibase.java

```diff
@@ -0,0 +1,186 @@
+/**
+*  This program is free software; you can redistribute it and/or
+*  modify it under the terms of the GNU General Public License
+*  as published by the Free Software Foundation; either version 3
+*  of the License, or (at your option) any later version. Additionally
+*  this file is subject to the "Classpath" exception.
+*
+*  This program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with this program; if not, write to the Free Software Foundation,
+*  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+package org.wikibase;
+
+import java.io.IOException;
+
+import java.net.URLEncoder;
+import java.util.StringTokenizer;
+import java.util.regex.Pattern;
+
+import org.wikibase.data.Entity;
+import org.wikipedia.Wiki;
+
+public class Wikibase extends Wiki {

```

Comment:
Please add a documentation comment that states the Wikibase repository extension is required to be installed on the wiki. Add a link to wikiba.se .